packages/flutter/test/material/tabs_test.dart

```diff
@@ -937,6 +939,262 @@ void main() {
     expect(find.text('Second'), findsNothing);
   });
 
+  testWidgets('TabBar animationDuration sets indicatior animation duration', (WidgetTester tester) async {
+    const Duration animDuration = Duration(milliseconds: 100);
+    final List<String> tabs = <String>['A', 'B', 'C'];
+
+    await tester.pumpWidget(buildFrame(tabs: tabs, value: 'B', animationDuration: animDuration));
+    final TabController controller = DefaultTabController.of(tester.element(find.text('A')))!;
+
+    await tester.tap(find.text('A'));
+    await tester.pump();
+    expect(controller.indexIsChanging, true);
+    await tester.pump(const Duration(milliseconds: 50));
+    await tester.pump(animDuration);
+    expect(controller.index, 0);
+    expect(controller.previousIndex, 1);
+    expect(controller.indexIsChanging, false);
+
+    //Test when index diff is greater than 1
+    await tester.pumpWidget(buildFrame(tabs: tabs, value: 'B', animationDuration: animDuration));
+    await tester.tap(find.text('C'));
+    await tester.pump();
+    expect(controller.indexIsChanging, true);
+    await tester.pump(const Duration(milliseconds: 50));
+    await tester.pump(animDuration);
+    expect(controller.index, 2);
+    expect(controller.previousIndex, 0);
+    expect(controller.indexIsChanging, false);
+  });
+
+  testWidgets('TabBarView controller sets animation duration', (WidgetTester tester) async {
+    const Duration animDuration = Duration(milliseconds: 100);
+
```

Comment:
```dart
const List<String> tabs = <String>['A', 'B', 'C'];
const List<String> views = <String>['0', '1', '2'];
```

This is just a trivial suggestion (along with the related ones below). Could be applied on several tests.